packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -511,6 +534,29 @@ describe('postgresql', () => {
                                                                         `)
     expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
+
+  test('introspection --url with postgresql provider but schema has a sqlite provider should fail', async () => {
+    ctx.fixture('schema-only-sqlite')
+    expect.assertions(7)
+
+    try {
+      await DbPull.new().parse(['--url', setupParams.connectionString])
+    } catch (e) {
+      expect(e.code).toEqual(undefined)
+      expect(e.message).toMatchInlineSnapshot(
+        `The database provider found in --url (postgresql) is different from the provider found in the Prisma schema (sqlite).`,
```

Comment:
What is the error code of this?

Reply:
None, it's an error for the CLI, which does not have error codes defined. The error code from the introspection engine is useless here.